core/unix/os.c

```diff
@@ -5540,11 +5540,13 @@ convert_to_non_prace_syscall_number(int sysnum)
 #        endif
     }
     if (sysnum == SYS_pselect6) {
-#        if defined(X86) || defined(ARM)
+#        if defined(X86)
```

Comment:
Why not just do `#if defined(SYS_select)` or `#ifdef SYS_select` rather than, in effect, maintain a list of architectures that have `SYS_select`?

Reply:
Good idea. Done